### PR TITLE
Add a README file

### PR DESCRIPTION
**Pull request by AntoineBrunet**
This PR adds a more readable and informative README file for the repository.

- [x] Pull request has descriptive title
- [x] Pull request gives overview of changes
- [x] New code has inline comments where necessary
- [x] Appropriate documentation has been written
- [x] Relevant issues are linked to (e.g. `See issue #` or `Closes #`)

### CODE REVIEW

**Review by mshumko — APPROVED** [review]
Looks good.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+Installation
+------------
+IRBEM requires a Fortran compiler, and can be installed on most
+environments.
+
+Quick install procedure on Linux with gfortran:
+
+    git clone https://github.com/PRBEM/IRBEM.git
+	cd IRBEM
+	make OS=linux ENV=gnu64 install
```
Comment (mshumko) [inline README.md:33]: These instructions are missing the ```make OS=linux64 ENV=gnu64 all``` command. Also the `linux` should be `linux64` in the ```make...install``` command

**Review by mshumko — CHANGES_REQUESTED** [review]
While trying to install a fresh version of IRBEM, I was reminded about the ```make...alll``` command.

File: README.md (hunk 1 above)
Comment (AntoineBrunet) [reply README.md]: Yes thanks, I have corrected the Readme. It should now compile (after applying this PR and #5 ) out of the box when following the instructions.

**Review by mshumko — APPROVED** [review]
Sorry for the delay @AntoineBrunet  but the README looks good. I have issues compiling IRBEM on Linux, but that is a separate issue that I'll get to next week.

**Comment by AntoineBrunet** [comment]
Thanks @mshumko ! Don't worry about the delay, I don't have much time on my side either. I was not sure you were aware that every PR must be reviewed before pulling in main.

**Pull request merged**